### PR TITLE
Add application "Airsonic"

### PR DESCRIPTION
**Pull request by Gofannon**

### CODE REVIEW

**Comment by yalh76** [comment]
There is an error for the airsonic url:
https://github.com/YunoHost-Apps/airsonic_ynh

The app can be found at https://github.com/YunoHost/airsonic_ynh

But I think, the might be moved from 
https://github.com/YunoHost to https://github.com/YunoHost-Apps

**Comment by Gofannon** [comment]
@yalh76 is right, I made a mistake while transferring ownership and I don't have the rights to "move" it to the right place.

**Comment by yalh76** [comment]
Maybe @anmol26s can send you an invitation to move your app from
https://github.com/YunoHost to https://github.com/YunoHost-Apps

**Comment by Gofannon** [comment]
Issue created on the bugtracker to ask an admin to move it : https://github.com/YunoHost/issues/issues/1298

**Comment by Psycojoker** [comment]
> Issue created on the bugtracker to ask an admin to move it : YunoHost/issues#1298

Done

File: community.json [hunk 1]
```diff
@@ -44,6 +44,13 @@
         "state": "inprogress",
         "url": "https://github.com/YunoHost-Apps/agora_ynh"
     },
+    "airsonic": {
+        "branch": "master",
+        "level": 3,
```
Comment (kay0u) [inline community.json]: The level is fixed by the CI. And the CI test the app only if the app is tagged as "working"

**Pull request merged**